.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Test & Coverage
```

Comment:
I think we can think of a better file name. Maybe even split the two?

Reply:
renamed to code coverage